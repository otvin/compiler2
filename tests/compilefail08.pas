program compilefail08(output);
{validates that a local variable named the same as a parameter into a function is properly detected}

var q:integer;

function oops(a:integer):integer;
var
	a:real;
begin
	a:=a+7;
	oops:=a*2;
end;

begin
	q := oops(3);
end.
