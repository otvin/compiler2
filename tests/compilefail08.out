Symbol Redefined: a in file:tests/compilefail08.pas line:6     column:15 
